api/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.2.5",
-    "@neo4j/graphql": "^3.4.0",
```

Comment:
But then why are we dropping the version number if that's. It the solution

Reply:
Will change it back to 3.4.0

Reply:
3.5.0

Reply:
Ok. Will do